src/test/a11y/tests/media-routes.test.ts

```diff
@@ -111,15 +111,34 @@ describe('Accessibility - Media User Routes', () => {
             beforeEach(() => {
                 sinon.restore();
                 sinon.stub(PublicationService.prototype, 'getCaseByCaseNumber').resolves(null);
-                sinon.stub(PublicationService.prototype, 'getCaseByCaseUrn').resolves(null);
             });
 
             describe('with no input data', () => {
-                testAccessibility(url, '', true, { 'search-input': '' });
+                testAccessibility(url, '', true, { 'case-number': '' });
             });
+        });
 
-            describe('with invalid input data', () => {
-                testAccessibility(url, '', true, { 'search-input': 'Invalid case number' });
+        describe('Bulk Unsubscribe Confirmation Page', () => {
+            const url = '/bulk-unsubscribe-confirmation';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { 'bulk-unsubscribe-choice': '' });
+            });
+        });
+
+        describe('Pending Subscriptions Page', () => {
+            const url = '/pending-subscriptions';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { userId: '' });
```

Comment:
userId is not the right postBody, it is part of req.user. In fact this route does not have a postBody so no need to pass in anything